src/org/exist/start/start.config

```diff
@@ -102,7 +102,10 @@ extensions/exquery/lib/*                           always
 extensions/exquery/restxq/lib/*                    always
 extensions/indexes/lucene/lib/*                    always
 extensions/indexes/spatial/lib/*                   always
-extensions/modules/lib/*                           always
+extensions/modules/cqlparser/lib/*                 always
```

Comment:
when i build there is no such directory, which means our docker builds are failing.
see eXist-db/docker-existdb#55